osu.Game/Screens/Edit/Compose/Components/Timeline/DifficultyPointPiece.cs

```diff
@@ -139,4 +142,37 @@ protected override void LoadComplete()
             }
         }
     }
+
+    internal partial class SliderVelocityInspector : EditorInspector
+    {
+        [BackgroundDependencyLoader]
+        private void load()
+        {
+            EditorBeatmap.TransactionBegan += updateInspectorText;
+            EditorBeatmap.TransactionEnded += updateInspectorText;
+            updateInspectorText();
+        }
+
+        private void updateInspectorText()
+        {
+            InspectorText.Clear();
+
+            double[] sliderVelocities = EditorBeatmap.HitObjects.OfType<IHasSliderVelocity>().Select(sv => sv.SliderVelocity).OrderBy(v => v).ToArray();
+
+            if (sliderVelocities.Length < 2)
+                return;
+
+            double? modeSliderVelocity = sliderVelocities.GroupBy(v => v).MaxBy(v => v.Count())?.Key;
+            double? medianSliderVelocity = sliderVelocities[sliderVelocities.Length / 2];
+
+            AddHeader("Beatmap average velocity");
```

Comment:
why not just "Average velocity"?

Reply:
Calling it an "average" at all is a stretch. The median is not the average.

That said, probably better to do a stats inaccuracy than have to explain to users what the median is, I guess.

Reply:
i'll admit that i didn't notice the median was used instead. imo there's no point in using inaccurate wording, as it's not "simpler" but "misleading".

Reply:
If anyone notices and complains we may change. But I think it's not a big deal. I'm guessing this was 100% intentional (because the variables are named correctly, it's just user-facing copy that's different).

Reply:
It's 101% intentional